docs/how-to/deploy/deploy-to-production.mdx

```diff
@@ -132,3 +132,48 @@ To achieve high throughput and low latency:
 - **Disable Decision Debug Info**: In production, you can turn off decision debug information to enhance performance. Disable debug logging by adding `--env PDP_DEBUG=False` to the PDP command line.
 - **Optimize Base Policy**: Tailor the base policy to your specific use case. For example, disabling ReBAC checks (if not used) can speed up policy execution (this requires our tech support).
   With these optimizations, you can expect several thousand RPS with less than 10ms latency per request.
+
+
+## PDP Offline Mode
+
+The PDP relies on Permit's cloud for fetching configuration, loading policies, and keeping those up to date.
+Despite the high availability of Permit's cloud, there still might be cases when the PDP can't reach it.
+For extra resilience, the PDP can be configured to work in offline mode, where it will use a local backup of the configuration, data, and policies.
+
+### Using Offline Mode
+
+To enable the PDP offline mode, use the `PDP_ENABLE_OFFLINE_MODE` environment variable.
+You would also have to mount a volume or a host directory to the PDP container at `/app/backup`, where the backup files will be persisted.
+When PDP offline mode is enabled, the PDP will automatically enable OPAL's offline mode, and will use the backup files from the mounted volume.
+
+```sh
+docker run -it -p 7766:7000 -p 8181:8181 
+  --env PDP_API_KEY=<YOUR_API_KEY> \
+  --env PDP_ENABLE_OFFLINE_MODE=true \
+  -v <HOST_BACKUP_DIR_OR_NAMED_VOLUME>:/app/backup
+  permitio/pdp-v2:latest
+```
+
+### How it works
+When the PDP is started in offline mode, it will try to fetch the configuration from Permit's cloud as usual.
```

Comment:
I think you should add here that one successful connection is required (to get the initial data or to provide it manually)

Reply:
What do you mean by "provide it manually"?

Reply:
I actually say it at the end sentence of the paragraph:
```
Once PDP is up and running - it won't try to fetch the configuration again from cloud, unless it's restarted.
```
I'll rephrase that paragraph, and emphasize that's the case both when loaded from cloud or from backup

Reply:
Rephrased the paragraph - lmk if that's better